.gitignore

```diff
@@ -5,6 +5,11 @@
 
 /dist*
 /builds
+#atqa-framework
+/atqa-framework/output*
```

Comment:
Why do you need a `*` here? Do you ignore just `output` folder or also some `outputblabla`?

Reply:
`/atqa-framework/output` is enough for that

Reply:
sorry for confusing, we have to keep an output folder and ignore only content inside cause this folder defined in the configs for the report data

Reply:
So if you remove the folder, it can't create it? Sounds strange